nautobot/dcim/api/views.py

```diff
@@ -317,49 +316,49 @@ class DeviceTypeViewSet(CustomFieldModelViewSet):
 #
 
 
-class ConsolePortTemplateViewSet(CustomFieldModelViewSet):
+class ConsolePortTemplateViewSet(NautobotModelViewSet):
```

Comment:
These dcim component templates don't have notes. Do we want to add that to the models or change the viewsets?

```
<class 'nautobot.dcim.models.device_component_templates.ConsolePortTemplate'>
<class 'nautobot.dcim.models.device_component_templates.ConsoleServerPortTemplate'>
<class 'nautobot.dcim.models.device_component_templates.PowerPortTemplate'>
<class 'nautobot.dcim.models.device_component_templates.PowerOutletTemplate'>
<class 'nautobot.dcim.models.device_component_templates.InterfaceTemplate'>
<class 'nautobot.dcim.models.device_component_templates.FrontPortTemplate'>
<class 'nautobot.dcim.models.device_component_templates.RearPortTemplate'>
<class 'nautobot.dcim.models.device_component_templates.DeviceBayTemplate'>
```

Reply:
These will not have notes_url as their fields if they don't support notes. I think this is acceptable. I am not sure if the template should/shouldn't have notes though, @glennmatthews do you have a thought?

Reply:
There are no template views, I don't believe they would need/use notes.  I checked the API and confirmed no errors and no `notes_url` field on the templates.

Reply:
It would probably make sense for these models to support notes in the long term, but if things work fine for now I'm okay with not making that change right now.

Reply:
Yeah, I second not making changes. We would need to add the detail view for templates as well.